artpaint/paintwindow/StatusView.cpp

```diff
@@ -410,7 +409,7 @@ SelectedColorsView::Draw(BRect area)
 	SetHighColor(255, 255, 255, 255);
 	StrokeRect(foreground_rect);
 	foreground_rect.InsetBy(1, 1);
-	foreground_rect = foreground_rect & area;
+//	foreground_rect = foreground_rect & area;
```

Comment:
I commented it to have a quick workaround for this:

https://cloud.githubusercontent.com/assets/6434360/23097501/3c7c63ca-f5ea-11e6-8a3a-6bd7c6d28287.png

I guess it was there for a more efficient filling of the square for the foreground color (black). We might be able to restore the line after the status view is converted to the layout API.